dtests/src/test/java/io/snappydata/hydra/cluster/SnappyTest.java

```diff
@@ -1699,8 +1536,15 @@ protected void getSnappyJobsStatus(String snappyJobScript, File logFile) {
             }
             inputFile.close();
             for (String str : jobIds) {
-                ProcessBuilder pb = new ProcessBuilder(snappyJobScript, "status", "--lead", leadHost + ":8090", "--job-id", str);
-                snappyTest.executeProcess(pb, logFile);
+                File log = new File(".");
+                String dest = log.getCanonicalPath() + File.separator + "jobStatus_" + RemoteTestModule.getCurrentThread().getThreadId() + "_" + System.currentTimeMillis() + ".log";
+                File commandOutput = new File(dest);
+                String expression = snappyJobScript + " status --lead " + leadHost + ":" + LEAD_PORT + " --job-id " + str + " > " + commandOutput + " 2>&1 ; grep -e '\"status\": \"FINISHED\"' -e 'curl:' " + commandOutput + " | wc -l)\"";
+                String command = "while [ \"$(" + expression + " -lt  1 ] || [ \"$(" + expression + " !=  1 ] ; do rm " + commandOutput + " ;  touch " + commandOutput + "   ; done";
```

Comment:
this will not work for multiple matches. please replace it to
while [ \"$(" + expression + "   -le 0 ]